plugins/processors/reversedns/rdnscache_test.go

```diff
@@ -0,0 +1,115 @@
+package reversedns
+
+import (
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestSimpleReverseDNSLookup(t *testing.T) {
+	d := NewReverseDNSCache(1*time.Second, 1*time.Second, -1)
```

Comment:
Not too concerned about short-lived tests.

Reply:
The tests double as documentation for how to use your type, so you should always use it correctly.  It will also be appreciated if one runs `go test -count 10000`.  Finally, its so easy, just do it ;)